### DISCUSSION TITLE
신고 당한 게시글/댓글이 수정된 경우 기존 데이터 확인 불가한 문제 관련

### DISCUSSION THREAD

**Discussion by hodin030** [cca-ffodregamdi/running-hi-back#186] [Trouble Shooting]
신고 당한 게시글/댓글이 수정된 경우 신고 근거 파악 불가능 -> 따라서 신고 저장 시 신고 당한 게시글/댓글 데이터 같이 저장되도록 변경

댓글 신고 시: 댓글 내용 저장
게시글 신고 시: 논의 다시 해봐야함!!!!!!

**Comment by silverpoodle** [comment]
힝.
